background/messages/mCreateWindow.ts

```diff
@@ -20,10 +21,11 @@ const handler: PlasmoMessaging.MessageHandler<
   )
 
   const createWindowUrl = `${runtime.getURL(
-    "tabs/mCreateWindow.html"
-  )}?origin=${req.body.origin}&account=${req.body.account}&tabId=${
-    req.sender.tab.id
-  }`
+    "tabs/createWebauthn.html"
+  )}?tabId=${req.sender.tab.id}&user=${encodeURI(
+    req.body.user
+  )}&chalBase64Url=${req.body.chalBase64Url}&authAttach=${req.body.authAttach}`
```

Comment:
I think it should be okay because the authAttach is of [string-like type](https://github.com/MasterKale/SimpleWebAuthn/blob/e36d2459cd167b1476100a6746a771965008cb6c/packages/typescript-types/src/dom.ts#L347),
```typescript
export type AuthenticatorAttachment = "cross-platform" | "platform";
```

and I have also [handled the case](https://github.com/pilagod/waallet-extension/blob/03c4b001855e8f9da6ae5a3d255200cdf63f1e2e/contents/messages.tsx#L116-L118) where the user passes an incorrect string."
```typescript
  const authAttach =
    params.authAttach &&
    (params.authAttach.toString() === "platform" ||
      params.authAttach.toString() === "cross-platform")
      ? params.authAttach
      : ("cross-platform" as AuthenticatorAttachment)
```